src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Controllers/CohortController.cs

```diff
@@ -219,19 +218,44 @@ public async Task<IActionResult> AddDraftApprenticeshipOrRoute(string changeCour
             if (await HasStartDateOverlap(model))
             {
                 StoreDraftApprenticeshipState(model);
-                return RedirectToAction(nameof(DraftApprenticeshipOverlapOptions));
+                return RedirectToAction(nameof(DraftApprenticeshipOverlapAlert));
             }
 
             return await SaveDraftApprenticeship(model);
         }
 
-          
+        [HttpGet]
+        [Route("add/apprenticeship/overlap-alert", Name = RouteNames.CohortOverlapAlert)]
+        public IActionResult DraftApprenticeshipOverlapAlert(DraftApprenticeshipOverlapAlertRequest request)
+        {
+            var model = PeekStoredDraftApprenticeshipState();
+
+            var vm = new DraftApprenticeshipOverlapAlertViewModel
+            {
+                DraftApprenticeshipHashedId = request.DraftApprenticeshipHashedId,
+                StartDate = model.StartDate.Date.GetValueOrDefault(),
+                EndDate = model.EndDate.Date.GetValueOrDefault(),
+                Uln = model.Uln,
+                FirstName = model.FirstName,
+                LastName = model.LastName
+            };
+            return View(vm);
+        }
+
+        [HttpPost]
+        [Route("add/apprenticeship/overlap-alert")]
+        public IActionResult DraftApprenticeshipOverlapAlert(DraftApprenticeshipOverlapAlertViewModel viewModel)
+        {
+            return RedirectToAction(nameof(DraftApprenticeshipOverlapOptions), new { DraftApprenticeshipHashedId = viewModel.DraftApprenticeshipHashedId });
+        }
```

Comment:
Overlap-alert endpoint should be moved to OverlappingTrainingDateRequestController.cs post merge from master

Reply:
Done